src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -1156,10 +1156,11 @@ struct alignas(4) RawHeadingOutput : public MessagePayload {
  *        version 1.0).
  * @ingroup measurement_messages
  *
- * The measurements in this message have been corrected for sensor mounting
- * angle biases.
- *
- * See also @ref RawHeadingOutput.
+ * The HeadingOutput message behaves similarly to the RawHeadingOutput,
+ * however, if no biases have been set AND the message
+ * is enabled the output will ALWAYS be Invalid.
```

Comment:
This comment is no longer correct. If heading biases are not specified, this message will not be generated.